pylabrobot/filters/RetroFilterV4/RetroFilterV4.md

```diff
@@ -0,0 +1,23 @@
+# Retro Filter V4
+
+<img src="RetroFilterV4.png" width="800" />
+
+Designed for flow cytometry prep.
```

Comment:
shall i move this to the docs so it's on the website? i would add it under liquid handling in the user guide